### PR TITLE
Fix clippy lints

### PR DESCRIPTION
**Pull request by GuillaumeGomez**

### CODE REVIEW

File: src/chunk/chunk.rs [hunk 1]
```diff
@@ -10,6 +10,7 @@ pub enum Chunk {
     BlockHalf(Vec<Chunk>),   //Block without open bracket, temporary
     UnsafeSmart(Vec<Chunk>), //TODO: remove (will change generated results)
     Unsafe(Vec<Chunk>),
+    #[allow(clippy::upper_case_acronyms)]
```
Comment (sdroege) [inline src/chunk/chunk.rs:13]: Yeah I'm not really a fan of that new lint. I disabled it in all my repos.

**Review by sdroege — APPROVED** [review]

**Comment by sdroege** [comment]
@GuillaumeGomez 

```
Run cd gtk-rs && python3 generator.py --no-fmt --gir-path ../target/release/gir --gir-files-directory ../tests/gir-files/ && rm ../Cargo.* && cargo build
usage: generator.py [-h]
                    [--gir-files-directories GIR_FILES_PATHS [GIR_FILES_PATHS ...]]
                    [--gir-path GIR_PATH] [--yes] [--no-fmt]
                    [path [path ...]]
generator.py: error: unrecognized arguments: --gir-files-directory
```

I think something needs to be updated here

**Comment by GuillaumeGomez** [comment]
Remaining issue seems unrelated. I'll investigate what's going on. Merging this one in the meantime.

**Comment by MarijnS95** [comment]
> Remaining issue seems unrelated. I'll investigate what's going on.
```
Run cd gtk-rs && python3 generator.py --no-fmt --gir-path ../target/release/gir --gir-files-directories ../tests/gir-files/ && rm ../Cargo.* && cargo build
=> Regenerating crates...
=> Looking in path `.`
==> Regenerating "pangocairo/Gir.toml"...
The following error occurred: expected str, bytes or os.PathLike object, not list
Traceback (most recent call last):
```

This one? https://github.com/gtk-rs/gtk-rs/pull/385 :grimacing:

**Comment by GuillaumeGomez** [comment]
Absolutely! It's friday, please be kind with my tired brain. ;)

**Comment by MarijnS95** [comment]
@GuillaumeGomez No worries, not pushing you... Just hoping you weren't digging for the same issue when a PR was already open :grimacing:

**Comment by GuillaumeGomez** [comment]
No luckily I wasn't. haha

**Pull request merged**